app-portage/golop/golop-0.2.1.ebuild

```diff
@@ -1,31 +1,37 @@
-# Copyright 1999-2021 Gentoo Authors
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI="6"
+EAPI="8"
 EGO_PN=github.com/klausman/golop
+inherit go-module
 
-if [[ ${PV} = *9999* ]]; then
-	inherit golang-vcs
+DESCRIPTION="Pure Go re-implementation of genlop"
+HOMEPAGE="https://github.com/klausman/golop"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/klausman/golop"
 else
-	KEYWORDS="~amd64 ~riscv ~x86"
-	EGIT_COMMIT=v${PV}
 	SRC_URI="https://${EGO_PN}/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
-	inherit golang-vcs-snapshot
+	EGIT_COMMIT=v${PV}
+	KEYWORDS="~amd64 ~riscv ~x86"
 fi
-inherit golang-build
 
-DESCRIPTION="A pure Go re-implementation of genlop"
-HOMEPAGE="https://github.com/klausman/golop"
 LICENSE="Apache-2.0"
 SLOT="0"
-IUSE=""
-DEPEND=""
-RDEPEND=""
+
+BDEPEND=">=dev-lang/go-1.15.0"
+
+DOCS=(
+	README.md
+)
```

Comment:
I _think_ you could remove this, `einstalldocs` should install `README*` files anyway.